test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -733,9 +733,13 @@ TEST_F(AtenXlaTensorTest, TestSymEig) {
           auto xla_b = torch::symeig(xla_a, eigenvectors, upper);
           AllClose(std::get<0>(b), std::get<0>(xla_b), /*rtol=*/3e-2,
                    /*atol=*/1e-2);
-          AllClose(std::get<1>(b).abs(), std::get<1>(xla_b).abs(),
-                   /*rtol=*/3e-2,
-                   /*atol=*/1e-2);
+          if (eigenvectors) {
```

Comment:
Hmm, why this?
Shouldn't pytorch return the same Zero we are returning now?
If not, what is the point of making the Zero change?

Reply:
PyTorch returns a 0-numel empty Tensor, which doesn’t have a value.

In ATen, we return `at::empty({0})`.

Reply:
Ouch!
Sorry to have misled you ... too many things going on.
Your previous code was correct ... brain *art on mine, sorry again.

Reply:
No problem, thanks! I've learned something new :-).

I'll revert the changes to `symeig.cpp`.

Reply:
Do we still need custom handling here?
Shouldn't comparing two tensors with a 0dim be returning equality?

Reply:
0-dim tensors could be scalars or 0-numel tensors and comparisons become weird. For example:

`torch.tensor([]) == torch.tensor(0)` is `torch.tensor([], dtype=torch.bool`